pkg/kubectl/resource/selector.go

```diff
@@ -50,11 +50,10 @@ func (r *Selector) Visit(fn VisitorFunc) error {
 	if err != nil {
 		if errors.IsBadRequest(err) || errors.IsNotFound(err) {
```

Comment:
nevermind... forgot about requestInfo not knowing about all possible API groups... this looks ok for now